docs/docs/admin/configuration/expressions.mdx

```diff
@@ -100,7 +100,7 @@ For this rule, if a request comes in matching [the signature of the `go get` com
 Anubis exposes the following variables to expressions:
 
 | Name            | Type                  | Explanation                                                                                                                               | Example                                                      |
-| :-------------- | :-------------------- | :---------------------------------------------------------------------------------------------------------------------------------------- | :----------------------------------------------------------- |
+|:----------------|:----------------------|:------------------------------------------------------------------------------------------------------------------------------------------|:-------------------------------------------------------------|
```

Comment:
So I can replicate the settings in VS Code, how did you format this?

Reply:
Ahhh it's just an option in goland. The table has an option to format

Reply:
I did find https://marketplace.visualstudio.com/items?itemName=darkriszty.markdown-table-prettify though